### PR TITLE
Simulator should use wait events rather than the wait object

### PR DESCRIPTION
**Pull request by rowanseymour**
Trying to get rid of wait objects on sessions because it's just duplication of what is in the wait event. Mailroom doesn't use them.

### CODE REVIEW

File: src/components/simulator/LogEvent.tsx [hunk 1]
```diff
@@ -72,6 +72,8 @@ export interface EventProps {
   classifier?: { uuid: string; name: string };
   ticketer?: { uuid: string; name: string };
   ticket?: { topic: Topic; body: string };
+  hint?: Hint;
+  timeout_seconds?: number;
```
Comment (rowanseymour) [inline src/components/simulator/LogEvent.tsx:76]: it'd be pretty neat if the simulator simulated timeouts and you could see a timer counting down to when you get moved on

File: src/components/simulator/Simulator.tsx [hunk 2]
```diff
@@ -131,7 +131,6 @@ interface Run {
   flow_uuid: string;
   status: string;
   events?: EventProps[];
-  wait?: Wait;
```
Comment (rowanseymour) [inline src/components/simulator/Simulator.tsx:134]: runs have never had waits

File: src/components/simulator/Simulator.tsx [hunk 3]
```diff
@@ -145,8 +144,7 @@ interface Session {
   runs: Run[];
   contact: Contact;
   input?: any;
-  wait?: Wait;
-  status?: string;
```
Comment (rowanseymour) [inline src/components/simulator/Simulator.tsx:149]: status is always set on sessions

File: src/components/simulator/LogEvent.tsx (hunk 1 above)
Comment (ericnewcomer) [reply src/components/simulator/LogEvent.tsx:76]: We talked about this in the past, but giving the user the option to trigger no-response. Could do both.

**Review by ericnewcomer — APPROVED** [review]

**Pull request merged**